docs/_sources/about_reco.rst.txt

```diff
@@ -107,3 +107,18 @@ Improving the highest-ranked recommendations has a more important effect than im
 
 .. math::
     NDCG@k = \frac{1}{\left | A \right |} \sum_{i=1}^{\left | A \right |} \frac {\sum_{r=1}^{\left | P_i \right |} \frac{rel(P_{i,r})}{log_2(r+1)}}{\sum_{r=1}^{\left | A_i \right |} \frac{1}{log_2(r+1)}}
+
+Diversity Recommender Metrics
+-----------------------------
+Diversity recommender metrics evaluate the quality of recommendations in terms of the diversity of recommended items.
```

Comment:
Is this sentence saying the same thing twice? : )

_"..Diversity metrics evaluate the quality of recommendations in terms of the diversity.."_

How about sth like:

"Diversity recommender metrics evaluate the quality of recommendations for different notions of diversity."

(and then in the next section, Inter-List Diversity is one such diversity notion..)